src/View/Helper/FormHelper.php

```diff
@@ -260,30 +260,30 @@ public function widgetRegistry(WidgetRegistry $instance = null, $widgets = [])
      */
     protected function _addDefaultContextProviders()
     {
-        $this->addContextProvider('orm', function ($request, $data) {
+        $this->addContextProvider('orm', function ($request, $data, $requestType) {
```

Comment:
Even then it won't work since `$this->requestType` is only set after `create()` is called and `_addDefaultContextProviders()` is called from constructor.